integration-libs/cdc/root/service/cdc-js.service.ts

```diff
@@ -239,18 +261,23 @@ export class CdcJsService implements OnDestroy {
   /**
    * Trigger CDC User log in using CDC APIs.
    *
-   * @param response
+   * @param email
+   * @param password
+   * @param context (optional) - indicates the user flow
    */
   loginUserWithoutScreenSet(
     email: string,
-    password: string
+    password: string,
+    context?: string
   ): Observable<{ status: string }> {
     return new Observable<{ status: string }>((isLoggedIn) => {
       (this.winRef.nativeWindow as { [key: string]: any })?.[
         'gigya'
       ]?.accounts?.login({
         loginID: email,
         password: password,
+        ...(context) && { context: context },
```

Comment:
Could you please run Prettier on this file too?

Reply:
Sure, will run yarn `prettier:fix`